<launch>

<!--Play a recorded file -->
<!--node pkg="rosbag" name="play" type="play"  args="$(find provider_sonar)/bags/2014-07-18-20-59-31.bag"/-->
<node pkg="rosbag" name="play" type="play"  args="--clock $(find provider_sonar)/bags/2014-07-18-21-02-59.bag"/>
<!--node pkg="rosbag" name="play" type="play"  args=" $(find provider_sonar)/bags/2014-07-18-21-05-09.bag"/-->
<!--node pkg="rosbag" name="play" type="play"  args=" $(find provider_sonar)/bags/2014-07-18-21-11-13.bag"/-->


<!-- Load parameters for the scanline parser-->
<rosparam command="load" file="$(find provider_sonar)/config/scan_line_parser_config.yaml"/>

<!--Create a node parser -->
<node pkg="provider_sonar" type="micron_driver_scanline_parser" name="scanline_parser" output="screen" />

<!--Set the frame for the Sonar -->
 <node pkg="tf" type="static_transform_publisher" name="Sonar_to_Sparus" args="1.0 0.0 0.0 0.0 0.0 1.57 /sparus2 /Micron 100" />

<!--Launch RViz -->
<node pkg="rviz" type="rviz" name="rviz" args="-d $(find provider_sonar)/config/Rviz_config.rviz" />
  
</launch>
